### ISSUE TITLE
Feature request: increased limits

### ISSUE THREAD

**Issue by rr-** [rr-/Tomb1Main#12]
- [x] Internal game memory - changed from 3.5 MB to 16 MB
- [x] Items (moveables) - changed from 256 to 10240
- [ ] Items - refactor to use dynamic allocation
- [ ] Rooms - remove hard limit of 1024
- [ ] Rooms to draw - protect against potential crash
- [ ] Textures (blocked by #63)

**Comment by rr-** [comment]
Blocked by #63

**Comment by rr-** [comment]
Closing in favor of new tickets.
